From 6790e7309c069ca35550de4b7e45c6b322393e58 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Mon, 1 Apr 2019 19:58:01 +0900
Subject: [PATCH 794/909] arm64: dts: salvator-common: Enable IPMMU of SDHI2

commit 2b6b2967bfea76c66819c36dbb78af8e84096ef1 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch enables the IPMMU of SDHI2 used as an eMMC port on Salvator
board.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/salvator-common.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/salvator-common.dtsi b/arch/arm64/boot/dts/renesas/salvator-common.dtsi
index 45489ec9a1c2..878ed7caca77 100644
--- a/arch/arm64/boot/dts/renesas/salvator-common.dtsi
+++ b/arch/arm64/boot/dts/renesas/salvator-common.dtsi
@@ -828,6 +828,8 @@
 	pinctrl-1 = <&sdhi2_pins_uhs>;
 	pinctrl-names = "default", "state_uhs";
 
+	iommus = <&ipmmu_ds1 34>;
+
 	vmmc-supply = <&reg_3p3v>;
 	vqmmc-supply = <&reg_1p8v>;
 	bus-width = <8>;
-- 
2.17.1

